projects/rero/ng-core/src/lib/record/detail/detail.component.html

```diff
@@ -15,12 +15,9 @@
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 <div class="alert alert-danger" *ngIf="error">{{ error | translate }}</div>
```

Comment:
Is this line still valid?

Reply:
Yes it's still valid, because it displays an error when the user can access to the page, but the API is not properly responding.